### PR TITLE
add simple way to search within changed files

### PR DESCRIPTION
**Pull request by tnrich**
Fixes https://github.com/letmaik/vscode-git-tree-compare/issues/99

@letmaik if you could take a look at this I'd appreciate it! Thx! 

Here's how it works:
image

image

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -26,6 +26,8 @@ In bigger projects with many files it also provides **context**, it gives you a
 
 - Log output of all git commands run
 
+- Search within changed files (Git Tree Comparse: Search Changes)
```
Comment (letmaik) [inline README.md:29]:
```suggestion
- Search within changed files

```

Let's keep this list simple, features should be discoverable easily via the UI.

**Review by letmaik — APPROVED** [review]
Great work, I like the simplicity!

**Comment by tnrich** [comment]
https://github.com/letmaik/vscode-git-tree-compare/actions/runs/11846248019/job/33059548196 why did this fail?

File: README.md (hunk 1 above)
Comment (tnrich) [reply README.md]: Done!

**Comment by letmaik** [comment]
> @letmaik https://github.com/letmaik/vscode-git-tree-compare/actions/runs/11846248019/job/33059548196 why did this fail?

Fixed here: https://github.com/letmaik/vscode-git-tree-compare/commit/7c30699dc4a6ae078ddadfc4d67f58be03e3f5f4

**Pull request merged**